go.mod

```diff
@@ -7,8 +7,10 @@ require (
 	github.com/golangci/plugin-module-register v0.1.1
 	github.com/onsi/ginkgo/v2 v2.22.2
 	github.com/onsi/gomega v1.36.2
+	golang.org/x/text v0.24.0
 	golang.org/x/tools v0.32.0
 	k8s.io/apimachinery v0.32.3
+	sigs.k8s.io/controller-tools v0.17.3
```

Comment:
I think adding a dependency on controller tools is likely a mistake for us. The project is not as stable as K/K and doesn't always keep dependencies up to date in the same way that K/K would.

When we come to getting this project set up on K/K, having a dependency on a slower project will likely hinder us as and when we come to update Kube to new version.